packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableRowWrapperDemo.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
```

Comment:
I see the table example in Table.md, but don't see anything new under demos. Is this intended to be under demos, the table example, or both?

Demo navigation links:
<img width="288" alt="Screen Shot 2019-10-17 at 11 12 58 PM" src="https://user-images.githubusercontent.com/17481322/67063141-ad9ec380-f133-11e9-8b61-42d92ce85877.png">

Reply:
Ah, I just realized I should have mentioned "example" instead of "demo" in the PR, wasn't intending for anything to show up here. I'll update the PR to reflect this